services/frontend/pkg/revaconfig/config.go

```diff
@@ -9,7 +10,15 @@ import (
 )
 
 // FrontendConfigFromStruct will adapt an oCIS config struct into a reva mapstructure to start a reva service.
-func FrontendConfigFromStruct(cfg *config.Config) map[string]interface{} {
+func FrontendConfigFromStruct(cfg *config.Config) (map[string]interface{}, error) {
+
+	webURL, err := url.Parse(cfg.PublicURL)
+	if err != nil {
+		return nil, err
+	}
+	webURL.Path = path.Join(webURL.Path, "/external")
```

Comment:
Why do you need the leading slash in `"/external`? Just for easy readability? (I'm aware that it does no harm)